extensions/itunes/init.lua

```diff
@@ -281,6 +281,20 @@ function itunes.setPosition(p)
   return tell('set player position to '..p)
 end
 
+--- hs.itunes.getDuration()
+--- Function
+--- Gets the duration (in seconds) of the current song
+---
+--- Parameters:
+---  * None
+---
+--- Returns:
+---  * The number of seconds long the current song is, 0 if no song is playing
+function itunes.getDuration()
+  local duration = tonumber(tell('duration of current track'))
+  return duration ~= nil and duration or 0
```

Comment:
Just my 2c but would it be better ti return `nil` instead of `0` ? Also what if the song is actually at `0`. Just something that came to mind...